components/kyma-environment-broker/internal/process/deprovisioning/initialisation.go

```diff
@@ -124,7 +129,29 @@ func (s *InitialisationStep) checkRuntimeStatus(operation internal.Deprovisionin
 
 	switch status.State {
 	case gqlschema.OperationStateSucceeded:
-		return s.operationManager.OperationSucceeded(operation, msg)
+		{
+
+			// TODO:
```

Comment:
is it still valid?

Reply:
We will do it as a follow up, so I removed it